packages/server/src/graphql/middlewares/measurables.js

```diff
@@ -64,8 +67,26 @@ async function measurableNameValidation(root, args, _context, _info) {
   return true;
 }
 
+/**
+ * @param {*} root
+ * @param {object} args
+ * @param {Schema.Context} context
+ * @param {object} _info
+ * @return {Promise<boolean>}
+ */
+async function measurableStateValidation(root, args, context, _info) {
```

Comment:
I don't like this name, and am confused by what it is validating. Is it validating that it is open to new non-comment measurements? If so, it should be renamed something like, ``measurableCanBeMeasuredValidation`` or something else specific.

Reply:
Yes, something wrong is in this function name. I am not a native English speaker so this is why sometimes it could have a place. This function should answer the question "Is the state of measurable in a right position for adding new measurement?". This is why I like this proposal "measurableCanBeMeasuredValidation".

Reply:
Ok, sounds good